xml/cl.xml

```diff
@@ -252,6 +252,7 @@ server's OpenCL/api-docs repository.
         <type category="define">typedef <type>cl_bitfield</type>      <name>cl_device_fp_atomic_capabilities_ext</name>;</type>
         <type category="define">typedef <type>cl_uint</type>          <name>cl_image_requirements_info_ext</name>;</type>
         <type category="define">typedef <type>cl_bitfield</type>      <name>cl_platform_command_buffer_capabilities_khr</name>;</type>
+        <type category="define">typedef <type>cl_bitfield</type>      <name>cl_mutable_dispatch_promises_khr</name></type>
```

Comment:
spec says `cl_mutable_dispatch_asserts_khr` not `cl_mutable_dispatch_promises_khr`